assets/css/main.scss

```diff
@@ -11,6 +11,12 @@ $overview-color: $bg-color;
 $agenda-color: $bg-color;
 $keypoints-color: #1D243D;
 $tip-color: #587177;
+
+$warning-color: #FAA632;
+$info-color: #4aafcd;
+$danger-color: #da4f4a;
+$success-color: #5ab75c;
```

Comment:
I personally really love these more colourful boxes! 

However, in the interest of keeping the tutorials looking uniform, I think we should try to minimize the number of different types of boxes we have, and instead re-use the existing box types where possible, and then maybe just rethink the colour palette/icon set for these at some point? For instance, perhaps we can re-use the `comment` box instead of the new `info` (and maybe `warning`) box to give the extra/background information to users? What do you think? 

Also, it looks like you don't use the `success` and `danger` boxes in your tutorial, could we remove those from the template for now?